src/Sarif.Multitool/Rules/SARIF2001.TerminateMessagesWithPeriod.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+
+using Microsoft.Json.Pointer;
+
+namespace Microsoft.CodeAnalysis.Sarif.Multitool.Rules
+{
+    public class TerminateMessagesWithPeriod : SarifValidationSkimmerBase
+    {
+        /// <summary>
+        /// SARIF2001
+        /// </summary>
+        public override string Id => RuleId.TerminateMessagesWithPeriod;
+
+        /// <summary>
+        /// Write in complete sentences and end each sentence with a period.This guidance
```

Comment:
>guidance [](start = 81, length = 8)

Formatting nit: Make the lines just even enough so that no complete word extends past the end.